database/migrations/2021_08_17_024713_create_parent_categories_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateParentCategoriesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('parent_categories', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
```

Comment:
[May] ER図では最大文字列長を指定しているので、マイグレーションに反映してほしい。他の nameカラムも同様に。
```suggestion
            $table->string('name', 45);
```